i18n/en.json

```diff
@@ -2412,6 +2412,18 @@
     "translation": "[{{ .SiteName }}] Your password has been updated"
   },
   {
+    "id": "api.templates.user_access_token_body.info",
+    "translation": "A user access token was added to your account on {{ .SiteURL }}. They can be used by integrations to access {{.SiteName}} with your account.<br>If this change wasn't initiated by you, please contact your system administrator."
```

Comment:
@jasonblais only comment is that `by integrations` is not necessarily true. If I have your user access token I can access your account as you without any integration set up